cardano-db/src/Cardano/Db/Insert.hs

```diff
@@ -214,7 +214,7 @@ insertManyRewardRests ::
   (MonadBaseControl IO m, MonadIO m) =>
   [RewardRest] ->
   ReaderT SqlBackend m ()
-insertManyRewardRests = insertManyCheckUnique "Many Rewards Rest"
+insertManyRewardRests = void . insertMany' "Many Rewards Rest"
```

Comment:
Is it worth catching any failures?

Reply:
It cannot fail due to a unique key, since it was removed. So I thjink it's fine